frontend/src/components/ForgotPasswordLayout.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import '../pages/forgotPassword/forgotPassword.css';
```

Comment:
you should keep styles by the component and use specific styles file for that component